nebula.config.js

```diff
@@ -12,5 +12,25 @@ module.exports = {
   },
   serve: {
     keyboardNavigation: false,
+    themes: [
+      {
+        id: 'Test',
+        theme: {
+          straightTable: {
+            header: {
+              fontSize: '40px',
+              color: 'cyan',
+              padding: '10px',
```

Comment:
Speros team wants to have the option https://github.com/qlik-oss/sn-table/pull/441. They found that the default padding is too big.
As far as I know, it will only work for sn-table not straight-table.

Reply:
Yes, but that is through options not through the theme

Reply:
Right, as far as I know, they only need to have the padding from Properties, not from the Sense Theme. 
But at the same time, I see that it is a bonus to add more support for the theme and no harm.

Reply:
Except that everything in the Theme should also be supported in the old table

Reply:
Alright, we should not give users wrong expectations, I gonna remove them